lib/functoria/install.mli

```diff
@@ -31,7 +31,7 @@ val empty : t
 val pp : t Fmt.t
 (** Print the .install rules to install [t] *)
 
-val pp_opam : t Fmt.t
+val pp_opam : ?subdir:Fpath.t -> unit -> t Fmt.t
 (** Print the opam rules to install [t] *)
```

Comment:
Extending the comment to explain the `~subdir` parameter would be useful

Reply:
done in 7148bba9